### PR TITLE
Add: playwright support of web testing

### PR DESCRIPTION
**Pull request by nereuseng**
In this PR, we

1. Added new end-to-end testing library Playwright
2. Added the capability of testing different types of browser, eg: Chromium, Firefox 

**How to test this feature:**
1. At `lib/web/autokin-web.js`, uncomment the part of playwright and comment the part of puppeteer.
2. Created the feature test case under `features/first.feature`
3. Paste the test case below:
```
@autokin 
Feature: My Feature
	As Autokin developer
	I want to verify that all API are working as they should

Scenario: My First Scenario
    Given that I use "firefox"
    Given that I have access to "someURLThatCanLogin"
    When I navigate to "/"
    Then I type "EMAIL" in field "#login-email-field"
    Then I type "PASSWORD" in field "#login-password-field"
    Then I click on "#button_login"
```

That should work and all steps should pass.

Please let me know if any help needed!

### CODE REVIEW

**Review by ariesbe — APPROVED** [review]

File: lib/web/autokin-playwright-builder.js [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+        }
+
+        let mockConfig = fs.readFileSync(target);
+        await this.mock(mockConfig);
+    }
+
+    async setBrowserType(browser) {
+        this.currentBrowser = browser;
+    }
+    // TODO 27th Jan: add type and click feature
```
Comment (nereus-fp) [inline lib/web/autokin-playwright-builder.js:189]:
```suggestion
```

Comment (nereuseng) [inline lib/web/autokin-playwright-builder.js:189]:
```suggestion
```

File: lib/web/autokin-playwright-builder.js [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+        this.hostUrl = Store.sanitize(hostUrl);
+        this.browser = await this.browserType[this.currentBrowser].launch();
+        this.page = await this.browser.newPage({
+            recordVideo: {
+                dir: 'videos/',
+                size: { width: 800, height: 600 }
+            }
+        });
+
+        // TODO should try without self
```
Comment (nereuseng) [inline lib/web/autokin-playwright-builder.js:29]:
```suggestion
```

**Comment by nereuseng** [comment]
@ariesb Could you help me to merge this PR if you are available? 😄

**Review by ariesb — APPROVED** [review]

**Review by ariesbe — APPROVED** [review]

**Pull request merged**